### PR TITLE
#4 : Create the home page lander

### LINKED ISSUE #4: 🎨  Create the home page lander
#  Create the home page lander
---

#### Note: This issue is dependent on #10, please complete that first.

Quick links: [Summary](#summary) | [Description](#description) | [Technical-Analysis](#Technical-Analysis) | [Acceptance-Criteria](#Acceptance-Criteria)

---

### Summary

As a user I want a nice home page that captures my attention.
I want a good experience on web and phone

### Description

Design:

image

image

Link to full design:
https://xd.adobe.com/view/7b72572b-b40a-4618-bfa6-5500ab28f13b-2e21/screen/63e63506-4ea3-4d02-af28-f7cf148b47d9

### Technical Analysis

This feature can be found in libs/pages/home
This feature should be a stand alone component. The page should be different from the element.

### Acceptance Criteria

1. When rendered the component is pixel perfect compared to the design above.
2. The component is clean and does not have more than 30 lines
3. The component SCSS uses CSS variables for colour, font, ... Connected to the main style.scss
4. The code is isolated to the libs/page/home package, unless a justified reason is specified.
5. The component is aligned using CSS flex
6. The design is responsive for screen, tablet and phone

### PR DESCRIPTION
**Pull request by Dcolonel6**
# Description
 Create the home page lander that captures user attention and gives a good experience on the phone and web

To achieve this, we needed to:

```
- As a user I want a nice home page that captures my attention.
- I want a good experience on web and phone
```

Fixes # (4) -  [Fixes #4 Create the home page lander:](https://github.com/italanta/elewa-group/issues/4)

## Type of change
- [x] New feature (non-breaking change which adds functionality)

# Screenshot 

## Web
Screen Shot 2023-02-16 at 22 56 27

## Mobile
Screen Shot 2023-02-16 at 22 54 58

# Checklist:
- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: libs/pages/elewa/home/src/lib/components/elewa-home-page-lander/elewa-home-page-lander.component.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<elewa-group-elewa-hero 
+    [subtitle]="'THE WAY IN WHICH WE TRAIN IS OUTDATED'" 
+    [title]="'We are a group of companies which unlock true human potential.'" 
+    [backgroundImg]="'https://images.unsplash.com/photo-1595583946260-2eb883f5ce11?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=870&q=80'"
+    [footerText] ="'Irrespective of origin, nationality, gender, and belief.'" >
+</elewa-group-elewa-hero>
```
Comment (IanOdhiambo9) [inline libs/pages/elewa/home/src/lib/components/elewa-home-page-lander/elewa-home-page-lander.component.html:6]: The hero should be on the home page, not a separate component.

File: libs/pages/elewa/home/src/lib/features-elewa-home.module.ts [hunk 2]
```diff
@@ -9,18 +9,19 @@ import { HomeHeroSectionComponent } from './components/home-hero-section/home-he
...
-    LayoutModule,
-
-    HomeRoutingModule
-  ],
+  imports: [CommonModule, LayoutModule, HomeRoutingModule],
   declarations: [
     HomePageComponent,
     HomeHeroSectionComponent,
     HomeJobsSectionComponent,
+    ElewaHomePageLanderComponent,
```
Comment (IanOdhiambo9) [inline libs/pages/elewa/home/src/lib/features-elewa-home.module.ts:19]: No need to have this component..

**Review by IanOdhiambo9 — CHANGES_REQUESTED** [review]

**Comment by Dcolonel6** [comment]
@IanOdhiambo9  changes have been effected

**Review by LemmyMwaura — APPROVED** [review]
Good job.

**Comment by IanOdhiambo9** [comment]
This issue has been merged, left margin has been fixed in issue as per design.

**Pull request closed without merging**